platform/client-admin/src/forms/volunteer.ts

```diff
@@ -0,0 +1,67 @@
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import SimpleSchema from 'simpl-schema';
+
+const volunteerForm = new SimpleSchema({
+    firstName: {
+        type: String,
+        max: 120,
+        uniforms: {
+            readonly: true
```

Comment:
Readonly?

Reply:
We should be able to edit lastName and firstName?

Reply:
ah.. those are comming from the keycloak so there could be two approaches.

One - admin creates or edits those and then we ignore keycloak completely (preferred). It might require some modifications in client app

Reply:
Okay thanks. Should do the same for the username and email? If we do so, the create will be easy.